### DISCUSSION TITLE
Como será organizado o estilo do nosso site?

### DISCUSSION THREAD

**Discussion by LucasRiboli** [ICEI-PUC-Minas-PMV-ADS/pmv-ads-2023-2-e1-proj-web-t6-calculadorarescisao#21] [Ideas]
Como será feito, TailWind, Bootstrap na "raça"? Vamos definir nessa discussão

**Comment by fabiobarkoski** [comment]
Acho que o uso de bootstrap seria uma boa, mas ao mesmo tempo podemos tentar com css puro, já que nosso projeto não precisa ter um front muito complexo
